ui/src/components/search-result.tsx

```diff
@@ -1,149 +1,177 @@
+                    " text-md font-poppins font-medium"}>
+                    {text_part.content}
+                  </span>
+                )
+              })}
+            </span>
+          }
+          {props.type === ResultType.Message &&
+            <p className="bg-[#352C45] p-2 px-4 rounded-lg font-poppins leading-[28px] border-b-[#916CCD] border-b-2">
+              {props.content.map((text_part, index) => {
+                return (
+                  <span key={index} className={(text_part.bold ? 'font-bold text-white' : '') +
+                    " fony-[14px] font-regular"}>
+                    {text_part.content}
+                  </span>
+                )
+              })}
+            </p>
+          }
+        </p>
       </div>
-    );
+    </div>
+  );
 }
 
+export function getPlatformDisplayName(platform: Platform) {
+  // This converts the platform name to title case for display purposes.
+  // example input : "google drive"
+  // example output: "Google Drive"
+  let list = platform.split("");
+  let output = "";
+  console.log(list)
+  for (let i = 0; i < list.length; i++) {
+    if (i === 0 || list[i-1] === " ") {
+      list[i] = list[i].toUpperCase();
+    }
+    output += list[i];
+  }
+  return output;
+}
 
-function getFormattedTime (time: string) {
-    let date = new Date(time);
-    return date.toLocaleDateString('en-US', {month: 'short', day: 'numeric', year: 'numeric'});
+function getFormattedTime(time: string) {
+  let date = new Date(time);
+  return date.toLocaleDateString('en-US', { month: 'short', day: 'numeric', year: 'numeric' });
 }
 
 function getSmallIconByPlatform(platform: Platform) {
-    let classes = "inline mx-1 fill-[#A3A3A3]";
-    switch (platform) {
-      case Platform.Confluence:
-        return <FaConfluence className={classes}></FaConfluence>
-      case Platform.Slack:
-        return <FaSlack className={classes}></FaSlack>
-      case Platform.Drive:
-        return <FaGoogleDrive className={classes}></FaGoogleDrive>
-    }
+  let classes = "inline mx-1 fill-[#A3A3A3]";
+  switch (platform) {
+    case Platform.Confluence:
+      return <FaConfluence className={classes}></FaConfluence>
+    case Platform.Slack:
+      return <FaSlack className={classes}></FaSlack>
+    case Platform.GoogleDrive:
+      return <FaGoogleDrive className={classes}></FaGoogleDrive>
+  }
 }
 
-function getBigIconByPlatform (platform: Platform) {
-    let classes = "mt-2 mr-2 h-[40px] w-[40px]";
-    switch (platform) {
-      case Platform.Confluence:
-        return <img className={classes} src={Confluence}></img>
-      case Platform.Slack:
-        return <img className={classes} src={Slack}></img>
-      case Platform.Drive:
-        return <img className={classes} src={GoogleDrive}></img>
+
+function ProfilePic(profilePicture, platform, ResultType) {
+  /*
+  This function displays an image: either a profile picture or a logo.
```

Comment:
Then call it ResultImage.
params should be: platform, resultType, profilePicture
with profilePicture having a default value